### PR TITLE
minor styling adjustment

### PR DESCRIPTION
**Pull request by vladimir32pa**

### CODE REVIEW

File: app/sections/view-solution-capabilities/template.njk [hunk 1]
```diff
@@ -23,38 +23,67 @@
   <h3>Generated HTML</h3>
   <pre><code>
     {{- '
-    <div class="nhsuk-width-container" data-test-id="view-solution-capabilities">
-      <div class="nhsuk-grid-row">
+    <div class="nhsuk-width-container"
```
Comment (alibombaye) [inline app/sections/view-solution-capabilities/template.njk]: Looks like it's missing `>` here

File: app/sections/view-solution-capabilities/template.njk [hunk 2]
```diff
@@ -23,38 +23,67 @@
...
+              </li>
+              <li>
+                <label class="nhsuk-label">
+                Feature 3
+                </label>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
```
Comment (alibombaye) [inline app/sections/view-solution-capabilities/template.njk]: I don't think this should belong here `view-features` section should belong here

File: app/sections/view-solution-description/template.njk [hunk 3]
```diff
@@ -23,48 +23,101 @@
...
+          </div>
+          <div data-test-id="view-question-data-text-email-address">
+            <label class="nhsuk-label">
+            jim2@solution.com
+            </label>
+          </div>
+        </div>
+      </div>
+    </div>
+    ' }}
```
Comment (alibombaye) [inline app/sections/view-solution-description/template.njk]: Don't think this contact details section belongs here

File: app/sections/view-solution-capabilities/template.njk (hunk 2 above)
Comment (vladimir32pa) [reply app/sections/view-solution-capabilities/template.njk]: it should, it is capability list

**Pull request merged**